tests/keys/cert.test.ts

```diff
@@ -44,7 +44,7 @@ describe('cert', () => {
       },
     ));
     expect(exitCode).toBe(0);
-    const certStatus = JSON.parse(stdout).certChainPEM;
+    const certStatus = JSON.parse(stdout).cert;
```

Comment:
Is this the same thing as the cert chain PEM? As in we should have something that tests for the cert chain! Not just the root cert pem.

Reply:
i'm pretty sure `tests/keys/certchain.test.ts` applies certchain tests

Reply:
speaking of, the second part of the test in `tests/keys/certchain.test.ts` is no longer needed and wasn't correct in the first place anyways, as `const certChainStatus = JSON.parse(stdout).rootCertChainPem;` never existed on the `agent status` command

Reply:
the second part of the similar test in `cert.test.ts` is also wrong, but doesn't matter anyway now that that certificates have been removed from `agent status` command

Reply:
What do you suggest?

Reply:
just removing the second part of those tests, they were meant to test `agent status` printing the certificates, but that is no longer the intended behaviour so i've removed it

Reply:
Ok so is this resolved?

Reply:
yes